docs/develop/create-sso-office-add-ins-aspnet.md

```diff
@@ -172,7 +172,7 @@ If you chose "Accounts in this organizational directory only" for **SUPPORTED AC
     var retryGetAccessToken = 0;
 
     async function getGraphData() {
-        await getDataWithToken({ allowSignInPrompt: true, allowConsentPrompt: true, forMSGraphAccess: true });
+        await getDataWithToken();
```

Comment:
Why is the parameter being removed? It's still in the sample: line 23 of the file: https://github.com/OfficeDev/PnP-OfficeAddins/blob/b6ce411d2ed935df4e68ec2185a9164fd64bef77/Samples/auth/Office-Add-in-ASPNET-SSO/Complete/Office-Add-in-ASPNET-SSO-WebAPI/Scripts/HomeES6.js

Reply:
For the flow of the start and the order of explanation of these parameters, this works better. I realize the complete code is slightly different.

Reply:
If the options aren't passed in at this point, then they don't get passed to getAccessToken. I think we have to keep this parameter, but add a bullet just above the snippet saying that the parameter will be explained in a later step.